src/moonlight-server/runners/docker.cpp

```diff
@@ -196,6 +196,34 @@ void RunDocker::run(std::string_view session_id,
     logs::log(logs::info, "[DOCKER] Starting container: {}", docker_container->name);
     logs::log(logs::debug, "[DOCKER] Starting container: {}", *docker_container);
 
+    auto pause_handler = this->ev_bus->register_handler<immer::box<events::PauseStreamEvent>>(
+        [session_id, container_id, this](const immer::box<events::PauseStreamEvent> &pause_ev) {
+          if (std::to_string(pause_ev->session_id) == session_id) {
+            docker_api.pause_by_id(container_id);
+          }
+        });
```

Comment:
Don't handle `PauseStreamEvent` (and`ResumeLobbyEvent` below) . For single users this can potentially break apps and for lobbies it will pause an entire lobby when any participant exits which you definitely don't want to happen.